records/using-tekton-results-for-openshift-pipelines-observability.adoc

```diff
@@ -23,8 +23,9 @@ include::modules/op-results-cert.adoc[leveloffset=+2]
 include::modules/op-results-db.adoc[leveloffset=+2]
 include::modules/op-results-storage.adoc[leveloffset=+2]
 
-
+//Install and configure Tekton Results
 include::modules/op-installing-results.adoc[leveloffset=+1]
+include::modules/op-configuring-retention-policy-results.adoc[leveloffset=+2]
```

Comment:
Just questioning whether it actually makes sense to nest this under installing, if you're configuring it later

Reply:
Yes that makes more sence, I imagined something like "Install and configure" section, but since that is not implemented right now, it would make more sence to not nest it :+1: